cunumeric/deferred.py

```diff
@@ -1980,7 +2009,7 @@ def add_arguments(
             task.add_input(scalar.base)
 
     @staticmethod
-    def compute_strides(shape: NdShape) -> NdShape:
+    def compute_strides(shape: NdShape) -> tuple[int, ...]:
```

Comment:
Nitpick change here, `strides` is conceptually not the shape of a table, so I just wrote out the type in full (even though `NdShape` is an alias for `tuple[int, ...]`).

Reply:
:+1: